_data/locales/es.yml

```diff
@@ -46,7 +46,6 @@ tumbleweed_live_info: "Por favor, tenga en cuenta las limitaciones de las imáge
   \ favor utilice la [imagen de Tumbleweed](/tumbleweed)\n* Tienen una selección limitada\
   \ de paquetes de software y controladores, por lo que no puede considerarse un reflejo\
   \ preciso de si Tumbleweed funcionará en su hardware o no\n"
-kubic_desc: Contenedor como Plataforma de Servicio basado en Kubernetes sobre openSUSE
```

Comment:
This is a multiline key, you missed a few of those in translation files

Reply:
All fixed, thanks for the heads up :)